heareval/score.py

```diff
@@ -239,6 +239,28 @@ class EventBasedScore(SoundEventScore):
             "percentage_of_length": 0.5,
         },
     ),
+    "event_onset_50ms_fms": partial(
+        EventBasedScore,
+        name="event_onset_50ms_fms",
+        score="f_measure",
+        params={
+            "evaluate_onset": True,
+            "evaluate_offset": False,
+            "t_collar": 0.05,
+            "percentage_of_length": 0.5,
```

Comment:
Remove this. percentage_of_length is not used when evaluate_offset is false.

Reply:
Should we also remove from the existing 200ms fms?